integration_testing/features/admin/admin-bulk-export-users-classrooms.feature

```diff
@@ -7,8 +7,9 @@ Feature: Admin bulk export of users and classrooms
       And there are learners enrolled in classrooms and coaches assigned to them
 
   Scenario: Export a CSV file containing all users in the facility
-    When I click on "Export" button under *Import and export users* heading
+    When I click on *Export* button under *Import and export users* heading
     Then I see the loading indicator
-      And after some seconds I see the *Open/Save as* window, or the file 'users_<date>_<time>.csv' is automatically saved on my local drive, depending on the browser defaults # <date> is current date and <time> current time, for example users_20200420_194415.csv
-    Then I open the csv file
-      And users belong only to my facility, even if there are more facilities in the device
+      And I am able to open or save or the file 'users_<date>_<time>.csv'
+      # <date> is current date and <time> current time, for example users_20200420_194415.csv
+    When I open the CSV file
```

Comment:
ah, great I didn't know I could begin with another `When` in the same scenario

Reply:
It might not be completely kosher if we were using this in automated BDD testing, but since these are steps for manual testing, we do use several `When > Then` pairs in the same scenario.